org.OpenT2T.Sample.SuperPopular.Lamp/Philips Hue Light Bulb/ts/thingTranslator.ts

```diff
@@ -0,0 +1,108 @@
+
+declare function require(module: string): any;
```

Comment:
shouldn't this be an import, not a require? I am not a typescript expert, but I thought that's how you do it in typescript? See https://basarat.gitbooks.io/typescript/content/docs/quick/nodejs.html